recipe/meta.yaml

```diff
@@ -29,11 +29,9 @@ outputs:
       host:
         - htslib >=1.15  # [not win]
         - m2w64-htslib >=1.15  # [win]
-        - tiledb 2.20.*
```

Comment:
`tiledb` still needs to be in `host`. It should be `tiledb 2.21.*`

Reply:
This is 0.29, which corresponds to Core 2.20, and contains an important backported fix. VCF 0.30 will be available in a few hours.

Reply:
Thanks for the clarification and sorry for the confusion. I thought the [release notes for 0.29.1](https://github.com/TileDB-Inc/TileDB-VCF/releases/tag/0.29.1) said 2.21.0, but they said 2.20.1. The host requirement can be restored to `tiledb 2.20.*` then